### PR TITLE
Feat/add api monitor

### PR DESCRIPTION
**Pull request by danshuitaihejie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_REGION=us-east-1
 PRE_SHARED_KEY=
+MONITER_ENABLE=True
```
Comment (R0uter) [inline README.md:14]: don't forget to update the env sample file.

**Review by R0uter — APPROVED** [review]
Small heads up: update the .env before merge, others looks good to me.

File: README.md (hunk 1 above)
Comment (R0uter) [reply README.md:14]: another thing, typo `MONITER` should be `MONITOR`

Comment (danshuitaihejie) [reply README.md:14]: > another thing, typo `MONITER` should be `MONITOR`

Thank you for correcting a mistake in my low -level

**Pull request merged**